app/templates/dayview.html

```diff
@@ -1,64 +1,92 @@
+                event.title }}</p>
+            {% if attr.total_time_visible %}
+            <p class="total-time text-truncate fw-light my-0">
+                {{attr.total_time}}</p>
             {% endif %}
         </div>
-        <div class="schedule">
-            <div class="container times bg-primeary position">
-                {% if view == 'day'%}
-                    {% for hour in range(24)%}
-                        <div class="row bg-transparent hourmark">
-                        {% set hour = hour|string() %}
-                        {{hour.zfill(2)}}:00
-                        </div>
-                    {% endfor %}
-                {% endif %}
-            </div>
-            <div class="eventgrid">
-                {% for event, attr in events %}
-                    <div id="event{{event.id}}" class="text-truncate px-2 d-flex flex-column justify-content-evenly event" style="background-color: {{attr.color}}; grid-row: {{attr.grid_position}}; max-hight:1.5rem;">
-                        <p class="text-truncate my-0 {{ attr.title_size_class }}">{{ event.title }}</p>
-                        {% if attr.total_time_visible %}
-                            <p class="total-time text-truncate fw-light my-0">{{attr.total_time}}</p>
-                        {% endif %}
-                    </div>
-                {% endfor %}
-            </div>
-            <div class="container baselines bg-transparent">
-                {% for i in range(25)%}
-                    <div class="hourbar {{i}} row text-white border-bottom">---</div>
-                {% endfor %}
-            </div>
-            <div class="actiongrid">
-                
-                {% for event, attr in events %}
-                    <div class="d-flex flex-row justify-content-around align-items-end action-continer" style="grid-row: {{attr.grid_position}};">
-                        <a href="/edit/{{event.id}}" title="Edit event" class="action-icon"><img class="pb-1" src="{{ url_for('static', path='/images/icons/pencil.svg')}}" width="15em" height="15em"></a>
-                        <a href="/delete/{{event.id}}" title="Delete event" class="action-icon"><img class="pb-1" src="{{ url_for('static', path='/images/icons/trash-can.svg')}}" width="15em" height="15em"></a>
-                    </div>
-                {% endfor %}
-            </div>
+        {% endfor %}
+    </div>
+    <div class="container baselines bg-transparent">
+        {% for i in range(25)%}
+        <div class="hourbar {{i}} row text-white border-bottom">---</div>
+        {% endfor %}
+    </div>
+    <div class="actiongrid">
+
+        {% for event, attr in events %}
+        <div class="d-flex flex-row justify-content-around align-items-end action-continer"
+             style="grid-row: {{attr.grid_position}};">
+            <a href="/edit/{{event.id}}" title="Edit event"
+               class="action-icon"><img class="pb-1"
```

Comment:
Prefer to break on a new element. Currently the HTML in this commit is hard to read

Reply:
that's not my code again